src/common.h

```diff
@@ -439,7 +439,7 @@ typedef struct sf_private_tag
 	sf_count_t		datalength ;	/* Length in bytes of the audio data. */
 	sf_count_t		dataend ;		/* Offset to file tailer. */
 
-	int				blockwidth ;	/* Size in bytes of one set of interleaved samples. */
+	sf_count_t		blockwidth ;	/* Size in bytes of one set of interleaved samples. */
```

Comment:
@ codeowners
It seemed sensible to change the blockwidth attribute to be an `sf_count_t`, since most places that it is used throughout the codebase assign it to be the product of int32s. But this change does substantially alter the dimensionality of this struct. Does it seem like this is a breaking API change? Should we avoid doing this?

Reply:
I guess no, it is private struct, we use `SNDFILE*` as opaque pointer:

https://github.com/libsndfile/libsndfile/blob/7ed4773251c26653b5ee1a0815bfe8b213c14b54/include/sndfile.h#L357-L359